alphadia/search_step.py

```diff
@@ -268,9 +269,13 @@ def _parse_modifications(mod_str: str) -> list[str]:
             )
             spectral_library = multiplexing(spectral_library)
 
-        library_path = os.path.join(self.output_folder, SPECLIB_FILE_NAME)
-        logger.info(f"Saving library to {library_path}")
-        spectral_library.save_hdf(library_path)
+        if (
+            self.config["general"]["save_library"]
+            or self.config["general"]["save_mbr_library"]
```

Comment:
why does this depend on `self.config["general"]["save_mbr_library"]` ? Thats a bit odd

Reply:
nvm, so we can reload it